src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
+const employeeList = document.querySelector('ul');
```

Comment:
Ensure that the `employeeList` element exists in the DOM. If the element is not found, the subsequent operations will fail. Consider adding a check to confirm its existence before proceeding.